source/yats/include/yats/task_configurator.h

```diff
@@ -69,14 +69,14 @@ class task_configurator : public abstract_task_configurator
         std::get<index>(m_listeners).push_back(typename type::function_type(std::move(callable)));
     }
 
-    std::unique_ptr<abstract_task_container> construct_task_container(std::unique_ptr<abstract_connection_helper> helper) const override
+    std::unique_ptr<abstract_task_container> construct_task_container(std::unique_ptr<abstract_connection_helper> helper) override
```

Comment:
I think this should at least be commented. We cannot call `construct_task_container` or `construct_connection_helper` twice because the member is gone.